src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSessionDetailsPanel.tag

```diff
@@ -30,7 +30,7 @@
                 <div class="form-group">
                     <label class="col-sm-2 control-label">Closing time:</label>
                     <div class="col-sm-10">
-                        <p class="form-control-static">${feedbackSession.endTimeString}</p>
+                        <p class="form-control-static" id="end-time">${feedbackSession.endTimeString}</p>
```

Comment:
@whipermr5 How would I get the date in that format though? Right now the date is in a very different format.